core-web/apps/dotcms-ui/src/app/view/components/_common/dot-autocomplete-tags/dot-autocomplete-tags.component.scss

```diff
@@ -79,10 +79,12 @@
                         display: block;
                         overflow: hidden;
                         text-overflow: ellipsis;
+                        white-space: nowrap;
```

Comment:
The chips should not be full width, must be `fit-content`, one next to each other and wrap when no space like:

```
----------------------------------------
| tag 1 | | tag 2 | | tag 3 | | tag 4 |
| tag 5 | | tag 6 |
----------------------------------------
```